src/lightning_lite/accelerators/cuda.py

```diff
@@ -11,14 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import logging
 import os
 import warnings
 from contextlib import contextmanager
 from functools import lru_cache
 from typing import Dict, Generator, List, Optional, Set, Union
 
 import torch
+from lightning_utilities.core.rank_zero import rank_zero_info
```

Comment:
We want to use our import, which will set the correct rank

```suggestion
from lightning_lite.utilities.rank_zero import rank_zero_info
```